include/pmacc/types.hpp

```diff
@@ -23,6 +23,39 @@
 
 #pragma once
 
+#include <cupla/types.hpp>
+
+#ifndef PMACC_CUDA_ENABLED
+#   define PMACC_CUDA_ENABLED ALPAKA_ACC_GPU_CUDA_ENABLED
+#endif
+
+#if( PMACC_CUDA_ENABLED == 1 )
+/* include mallocMC before cupla renaming is activated, else we need the variable acc
+ * to call atomic cuda functions
+ */
```

Comment:
good idea to investigate, for that we need to implement alternatives for the few native asm calls.